keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -232,21 +236,12 @@ Additionally, a new `Pod.Spec.Containers[i].ResizePolicy[]` field (type
 
 When the Kubelet admits a pod initially or admits a resize, all resource requirements from the spec
 are cached and checkpointed locally. When a container is (re)started, these are the requests and
-limits used.
+limits used. Only the allocated requests are reported in the API, through the
+`Pod.Status.ContainerStatuses[i].AllocatedResources` field.
 
-The alpha implementation of In-Place Pod Vertical Scaling included `AllocatedResources` in the
-container status, but only included requests. This field will remain in alpha, guarded by the
-separate `InPlacePodVerticalScalingAllocatedStatus` feature gate, and is a candidate for future
-removal. With the allocated status feature gate enabled, Kubelet will continue to populate the field
-with the allocated requests from the checkpoint.
-
-The scheduler uses `max(spec...resources, status...resources)` for fit decisions, but since the
-actual resources are only relevant and reported for running containers, the Kubelet sets
-`status...resources` equal to the allocated resources for non-running containers.
-
-See [`Alternatives: Allocated Resources`](#allocated-resources-1) for alternative APIs considered.
-
-The allocated resources API should be reevaluated prior to GA.
+The scheduler uses `max(spec...resources, status...allocatedResources, status...resources)` for fit
```

Comment:
A note to myself: This is a joint decision we made together, but it could lead the resource under utilized in the worst scenario.